assets/Gruntfile.js

```diff
@@ -207,6 +208,8 @@ module.exports = function (grunt) {
           'bower_components/uri.js/src/URI.min.js',
           'bower_components/uri.js/src/jquery.URI.min.js',
           'bower_components/uri.js/src/URI.fragmentQuery.js',
+          // bower registration error? we get 2x versions of uri.js/urijs
+          'bower_components/urijs/',
```

Comment:
Sticking with using `uri.js` & opting out of `urijs`.  Is just messy trying to flip-flop them, really not sure why.  Something must be messed up with the way that repo was double registered (perhaps around the time of the version we are using?), but swapping generates a bunch of errors.